core/src/main/scala/spire/math/Interval.scala

```diff
@@ -421,6 +423,9 @@ sealed abstract class Interval[A](implicit order: Order[A]) { lhs =>
       case (_, _) => lhs * rhs.reciprocal
     }
 
+  def /(rhs: A)(implicit ev: Field[A]): Interval[A] =
+    lhs * rhs.reciprocal
```

Comment:
No worries, it's a good reminder that maybe I should take another crack at doing it right.